docs/api/command-line-switches.md

```diff
@@ -241,6 +241,12 @@ Force using discrete GPU when there are multiple GPUs available.
 
 Force using integrated GPU when there are multiple GPUs available.
 
+### --xdg-portal-required-version=`version`
+
+Sets the minimum required version of XDG portal implementation to `version`
+inorder to use the portal backend for file dialogs on linux. Current
+default is set to `3`.
```

Comment:
What's the behavior when the environment doesn't meet the required version?

Reply:
Missed documenting here, we fallback to the gtk or kde based file dialogs depending on the environment.